codewind-che-sidecar/deploy-pfe/pkg/constants/defaults.go

```diff
@@ -18,6 +18,9 @@ const (
 	// PFEImageTag is the image tag associated with the docker image that's used for Codewind-PFE
 	PFEImageTag = "latest"
 
+	// PFEVolumeSize is the size of the volume to use for PFE
+	PFEVolumeSize = "1Gi"
```

Comment:
Is `1Gi` enough as a default? Is that the same as Che?

Reply:
Should be fine as that's what we've been using, but I've opened https://github.com/eclipse/codewind/issues/956 to make this configurable